### PR TITLE
CCS content changes (Part 2)

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?

Content changes for the accommodation section of the CCS survey, this includes a new question, and new question variants throughout the section.

### How to review

Check that the changes match the 'Content 2' tab in the spreadsheet attached to the trello card.

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### England

- [CCS](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/ccs-content-changes-2/schemas/en/ccs_household_gb_eng.json)

### CODE REVIEW

**Comment by berroar** [comment]
[[I] CCS Corrections- Content 2 (S)](https://trello.com/c/6y6ajzGD/3889-i-ccs-corrections-content-2-s)

File: source/jsonnet/england-wales/ccs/blocks/accommodation/interviewer_note.jsonnet [hunk 1]
```diff
@@ -0,0 +1,29 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+
+{
+  type: 'Question',
+  id: 'interviewer-note',
+  question: {
+    id: 'interviewer-note-question',
+    title: {
+      text: 'Interviewer Note: Was the respondent living at {household_address}, on {census_date}?',
```
Comment (MebinAbraham) [inline source/jsonnet/england-wales/ccs/blocks/accommodation/interviewer_note.jsonnet:9]: According to sheets
```suggestion
      text: 'Interviewer Note: Was the respondent living at {household_address} on Sunday {census_date}?',
```

Comment (MebinAbraham) [reply source/jsonnet/england-wales/ccs/blocks/accommodation/interviewer_note.jsonnet:9]: Needs to update translation after also.

File: source/jsonnet/england-wales/ccs/blocks/accommodation/accommodation_type.jsonnet [hunk 2]
```diff
@@ -1,39 +1,48 @@
...
-          {
-            label: 'Caravan or other mobile or temporary structure',
-            value: 'Caravan or other mobile or temporary structure',
-          },
-        ],
-      },
-    ],
-  },
+    {
+      question: question('What type of accomodation was this?'),
```
Comment (petechd) [inline source/jsonnet/england-wales/ccs/blocks/accommodation/accommodation_type.jsonnet:42]: Should be "What type of accomodation was **that**?"

**Pull request merged**